src/cljc/quil/core.cljc

```diff
@@ -682,22 +682,21 @@
                   (float cx2) (float cy2) (float cz2)
                   (float x) (float y) (float z))))
 
-(defn
-  ^{:require-binding false
-    :processing-name "binary()"
-    :category "Data"
-    :subcategory "Conversion"
-    :added "1.0"}
-  binary
-  "Returns a string representing the binary value of an int, char or
-  byte. When converting an int to a string, it is possible to specify
-  the number of digits used."
-  ([val]
-   #?(:clj (PApplet/binary (int val))
-      :cljs (.binary (ap/current-applet) val)))
```

Comment:
We can use JS to implement it:

`(.toString val 2)`

and 

`(subs (.toString val 2) 0 num-digits)`

for the second one

Reply:
👍